### PR TITLE
feat: add quickstart synthetics/private-locations

### PR DESCRIPTION
**Pull request by keegoid-nr**
# Summary

- add dashboard json
- add config.yml with descriptions
- add screenshots 800px wide
- fix broken script link in CONTRIBUTING.md
- fix some markdown linting errors in CONTRIBUTING.md

Not sure what to do about the install plan. Any help is appreciated.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

synthetics-private-minions01
synthetics-private-minions02
synthetics-private-minions03
synthetics-private-minions04
synthetics-private-minions05

### CODE REVIEW

File: quickstarts/synthetics/private-locations/config.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+id:
```
Comment (josephgregoryii) [inline quickstarts/synthetics/private-locations/config.yml:1]: Hey @keegoid-nr  - We generate the ID automatically here after merging in a new quickstart:
https://github.com/newrelic/newrelic-quickstarts/blob/ed53333e26e9d8964fd3e245459f1c91d60fc0f3/utils/generate-uuids.js#L15-L20

We just need to remove this line from the config file. Other than that, everything looks great!

**Review by josephgregoryii — CHANGES_REQUESTED** [review]
Everything looks great, just need to remove one line in the `config.yml` file

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @keegoid-nr! Thanks so much for this contribution and the README updates!
I left a comment about adding an install plan, since you have a dashboard you'll need an install plan to make the quickstart "installable"

File: quickstarts/synthetics/private-locations/config.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - cpm
+  - private locations
+  - private location queue
+  - minion
+  - minion dashboard
+  - synthetics
+
```
[the remark this answers is not in the document]
Comment (keegoid-nr) [reply quickstarts/synthetics/private-locations/config.yml:49]: Thanks @aswanson-nr, I added an install plan based on your advice. Let me know if you see any issues with it.

File: quickstarts/synthetics/private-locations/dashboards/synthetics-private-minions/synthetics-private-minions.json [hunk 3]
```diff
@@ -0,0 +1,2463 @@
+{
+  "name": "Synthetics Private Minions",
+  "description": "Details on events from SyntheticsPrivateLocationStatus, SyntheticsPrivateMinion, SyntheticCheck, SyntheticRequest, NrAuditEvent, plus a page for Performance Analysis.",
+  "permissions": "PUBLIC_READ_WRITE",
```
Comment (aswanson-nr) [inline quickstarts/synthetics/private-locations/dashboards/synthetics-private-minions/synthetics-private-minions.json:4]: Just need to remove this permissions field
```suggestion
```

**Review by aswanson-nr — APPROVED** [review]

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**